### DISCUSSION TITLE
Mail function doesn't work

### DISCUSSION THREAD

**Discussion by cracksilver** [automadcms/discussions#46] [Q&A]
hi there
I moved my whole installation to another server. Since then I can't configure the mail function and send test mails.
I get only a message with a failure. 
![grafik](https://github.com/user-attachments/assets/ac98a070-3ca1-4ac1-89e5-d1c43f1b295f)

So is there somewhere else to see what exactly is the problem? I checked with SMTP and sendmail too. 

Any help would be appreciated.

**Comment by cracksilver** [comment]
I tried to send mail with a new blank installation and have the same problem. Can't understand cause the server send normal mails without any problems from this domain.

**Comment by cracksilver** [comment]
Now I checked the mail function on this server. It works perfect and sends me an email.

`<?PHP
$sender = ' test@mydomain.com ';
$recipient = ' receiver@anothertestdomain.com' ;

$subject = "php mail test";
$message = "php test message";
$headers = 'Von:' . $sender;

if (mail($recipient, $subject, $message, $headers))
{
    echo "Nachricht akzeptiert";
}
else
{
    echo "Fehler: Nachricht nicht akzeptiert";
}
?>`

Is there a logfile in automad where I can check what is going wrong in automad?

**Comment by marcantondahmen** [comment]
Did you add an email address to your user account? Test emails are always sent to a user email address.

**Comment by cracksilver** [comment]
yes they have both testsystems an email address in the user account.

**Comment by cracksilver** [comment]
logfile would be nice to see where automad is struggeling. On the old server it works fine with the exact same files. 

What about the debugin mode? If I turn it on, where is this called temp folder?

**Comment by marcantondahmen** [comment]
Debugging is described in the docs. What kind of server do you use and what PHP version?

**Comment by cracksilver** [comment]
![394223806-e7afecd9-7ca7-49a6-a068-69cb169e05ab](https://github.com/user-attachments/assets/d3ed6118-3666-4cdb-b2ee-70f008b9f463)
